controller:
  chassis_controller:
    type: chassis_controller/ChassisController
    pid_back_left_:
      p: 9.98
      i: 0.01
      d: 0.0005
    pid_back_right_:
      p: 9.98
      i: 0.01
      d: 0.0005
    pid_front_left_:
      p: 9.98
      i: 0.01
      d: 0.0005
    pid_front_right_:
      p: 9.98
      i: 0.01
      d: 0.0005
  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50